### PR TITLE
Updated python tool readme + added pip instructions

### PR DESCRIPTION
**Pull request by NicoHood**
Tested on ubuntu 16.04

### CODE REVIEW

File: tools/_python_framework/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+crypto
```
Comment (limpkin) [inline tools/_python_framework/requirements.txt]: did you test what you are trying to commit?

Comment (NicoHood) [inline tools/_python_framework/requirements.txt]: Didn't I already wrote that?

Comment (limpkin) [inline tools/_python_framework/requirements.txt]: where?

Comment (NicoHood) [inline tools/_python_framework/requirements.txt]: In the description of the PR

Comment (limpkin) [inline tools/_python_framework/requirements.txt]: I'm making sure... as it isn't the first time you'd try to push something not tested.

**Comment by limpkin** [comment]
So I tried your instructions on a brand new 16.04 and unsurprisingly it didn't work. Thanks for nothing.
Readme is updated with proper instructions.

**Comment by NicoHood** [comment]
What exactly did not work? You have not changed much, all I can see is that you added a `python-crypto` dependency. Maybe this was already on my system, but that doesnt mean my solution was wrong!? So can you give me a few more details please?

**Comment by NicoHood** [comment]
So now I got time to check the instructions again. It looks like this is the only relevant change you made:
https://github.com/limpkin/mooltipass/commit/8a12246706d4c75263c7aca6e64da33efd710b99#diff-25ec6de72bcdd7baf65e7f49d8c4ca4f

So I dont understand why you are so aggressive. Those instructions did work. They are just not "click here click there", as they were made for linux in general and not ubuntu. I just tested it with ubuntu.

Also you should still keep a general solution, as not all of use use ubuntu. And in some distibutions the packages have a trailing "2" and sometimes not.

On top of that is a very bad idea to use sudo to call the update proceedure. You should refer to the udev rule textfile and remove the sudo. Calling such scripts as root is dangerours, as so much things could go wrong. So its better to stay safe and only allow access to the MP but not the whole root filesystem etc.

The python command is also not required as we call python2 in the first line of the script. If the file is executeable (which it is) this should not be a problem. This is distribution independant. Also you can change what `python` points to on ubuntu and some users might have changed this to python3. So simply calling the script would be better, simpler and smaller.

I am fine with your ubuntu instructions but would still add a genral linux solutiona and the sudo fix. If you want I can prepare another PR. Not sure if you are fine with that, so let me know ;)

**Pull request merged**